staging/app/outputs.tf

```diff
@@ -1,3 +1,15 @@
 output "alb_hostname" {
   value = aws_alb.staging.dns_name
 }
+
+output "aws_security_group-ecs-tasks-staging" {
```

Comment:
We output some security group and subnet IDs that are important when running one-off tasks (db migrations, etc.) from the command line.